packages/zk-transfer-web/src/libs/txHelpers.ts

```diff
@@ -0,0 +1,82 @@
+import BN from "bn.js";
+import { Map } from "immutable";
+import { Demo, EthAddress, BM, SignerEth } from "./types";
+import { untilTxMined } from "demo-tx";
+import { runTransforms } from "demo-transform";
+import { deployed, constructMintPipeline } from "demo-aztec-lib";
+import { RINKEBY_ADMIN_ADDRESS, RINKEBY_ADMIN_PASSWORD } from "./constants";
+
+const mintPipeline = constructMintPipeline();
```

Comment:
we can probably do
```
import { deployed, mint } from "demo-aztec-lib";
```
and if `mint` isn't exported from`aztec-lib`, go ahead and export it.

Then you don't need to re-defined `mint` and `mintPipeline` here.

Reply:
It's `mintFunc` in the aztech-lib not `mint`